src/status_im2/contexts/communities/actions/airdrop_addresses/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.communities.actions.airdrop-addresses.view
+  (:require
+    [quo.components.list-items.account.view :as account-view]
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.common.not-implemented :as not-implemented]
+    [status-im2.contexts.communities.actions.airdrop-addresses.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- render-item
+  [item]
+  [account-view/view
+   {:account-props item
+    :emoji         (:emoji item)}])
+
+(defn- accounts-list
+  [{:keys [accounts]}]
+  [rn/view {:style style/account-list-container}
+   (when (seq accounts)
+     [rn/flat-list
+      {:data      accounts
+       :render-fn render-item
+       :key-fn    :id}])])
+
+(defn- airdrop-addresses
+  [{:keys [accounts logo-uri community-name]}]
+  [rn/view {:style style/airdrop-addresses}
+   [rn/view {:style style/airdrop-addresses-header}
+    [quo/text {:size :heading-2 :weight :semi-bold} (i18n/label :t/airdrop-addresses)]
+    [quo/button
+     {:type       :grey
+      :size       32
+      :icon-only? true
+      :on-press   #(not-implemented/alert)} 
+     :i/info]] 
+   [quo/context-tag
+    {:type            :community
+     :size            24
+     :community-logo  logo-uri
+     :community-name  community-name
+     :container-style {:margin-top 8}}]
```

Comment:
As you mentioned [here](https://github.com/status-im/status-mobile/pull/18180/files#r1427104905), we can use `drawer-top` component.